tests/unit/test_feed_eviction.py

```
@@ -158,3 +159,54 @@ def test_eviction_null_handling(client):
         # Verify a dated item (newest) is still there
         check_dated = FeedItem.query.filter_by(guid="guid-0").first()
         assert check_dated is not None
+
+
+def test_get_ids_to_evict_under_limit(client):
```

Ruff (0.15.2)

[warning] Unused function argument: `client`

(ARG001)